xml/hwbp_registers.xml

```diff
@@ -726,17 +732,17 @@
         </field>
 
         <field name="m" bits="9" access="WARL" reset="0">
-            When set, every instruction completed in or trap taken from
-            M-mode decrements \FcsrIcountCount by 1.
+            When set, enable this trigger in M-mode.
+        </field>
+        <field name="pending" bits="8" access="R/W" reset="0">
+            This bit becomes set when \FcsrIcountCount is decremented from 1
+            to 0. It is cleared when the trigger fired.
         </field>
-        <field name="0" bits="8" access="R" reset="0" />
         <field name="s" bits="7" access="WARL" reset="0">
-            When set, every instruction completed in or trap taken from
-            S/HS-mode decrements \FcsrIcountCount by 1.
+            When set, enable this trigger in S/HS-mode.
```

Comment:
Is there somewhere else that explains that both committed instructions and traps should decrement the count? I can't find that in this icount section.

Reply:
Nevermind; I see this explained on line 686 in the merged file.